Testscripts/Linux/LIS-MODULES-CHECK.py

```diff
@@ -32,6 +33,23 @@ def RunTest(command):
     if output:
         hvModules.remove("hid_hyperv")
 
+    [current_distro, distro_version] = DetectDistro()
+    lis_exists=Run("rpm -qa | grep hyper-v 2>/dev/null")
+
+    if distro_version.startswith("7") and lis_exists:
+        hvModules.append("pci_hyperv")
+        output = Run(command)
+        if not ("pci_hyperv" in output):
+            Run("modprobe pci_hyperv")
+
+    if (distro_version.startswith("7.3") or distro_version.startswith("7.4")) and lis_exists:
```

Comment:
you should not hardcode versions. at best put a greater than condition, if you know messages are to appear only from a version onwards.